src/test/java/unit/uk/gov/ida/verifyserviceprovider/factories/saml/AuthnRequestFactoryTest.java

```diff
@@ -1,32 +1,68 @@
 package unit.uk.gov.ida.verifyserviceprovider.factories.saml;
 
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.codec.binary.Base64;
 import org.junit.Before;
 import org.junit.Test;
+import org.opensaml.saml.saml2.core.Attribute;
 import org.opensaml.saml.saml2.core.AuthnRequest;
+import org.opensaml.saml.saml2.core.EncryptedAttribute;
+import org.opensaml.saml.saml2.core.Extensions;
+import org.opensaml.saml.saml2.encryption.Decrypter;
+import org.opensaml.saml.saml2.encryption.Encrypter;
+import org.opensaml.security.credential.BasicCredential;
+import org.opensaml.xmlsec.encryption.support.DecryptionException;
+import uk.gov.ida.common.shared.security.PrivateKeyFactory;
+import uk.gov.ida.common.shared.security.PublicKeyFactory;
+import uk.gov.ida.common.shared.security.X509CertificateFactory;
 import uk.gov.ida.saml.core.IdaSamlBootstrap;
+import uk.gov.ida.saml.core.extensions.versioning.Version;
 import uk.gov.ida.saml.core.test.PrivateKeyStoreFactory;
 import uk.gov.ida.saml.core.test.TestEntityIds;
+import uk.gov.ida.saml.security.DecrypterFactory;
 import uk.gov.ida.verifyserviceprovider.dto.LevelOfAssurance;
+import uk.gov.ida.verifyserviceprovider.factories.EncrypterFactory;
 import uk.gov.ida.verifyserviceprovider.factories.saml.AuthnRequestFactory;
+import uk.gov.ida.verifyserviceprovider.utils.ManifestReader;
 
 import java.net.URI;
+import java.security.PrivateKey;
+import java.security.PublicKey;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.HUB_TEST_PRIVATE_ENCRYPTION_KEY;
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.HUB_TEST_PUBLIC_ENCRYPTION_CERT;
 
 public class AuthnRequestFactoryTest {
 
     private static final URI DESTINATION = URI.create("http://example.com");
     private static final String SERVICE_ENTITY_ID = "http://entity-id";
-
-    private static AuthnRequestFactory factory = new AuthnRequestFactory(
-        DESTINATION,
-        new PrivateKeyStoreFactory().create(TestEntityIds.TEST_RP).getSigningPrivateKey()
-    );
-
+    private static final ManifestReader manifestReader = mock(ManifestReader.class);
+    private static final EncrypterFactory encrypterFactory = mock(EncrypterFactory.class);
+    private static Encrypter encrypter;
+    private static Decrypter decrypter;
+    private static AuthnRequestFactory factory;
 
     @Before
     public void bootStrapOpenSaml() {
```

Comment:
This method probably wants renaming now it's doing more than just bootstrapping. Happy with `before` or `setUp` - whatever the Cool Kids are calling their `@Before` methods these days.